ui/analyse/css/explorer/_config.scss

```diff
@@ -117,10 +117,26 @@
     }
     .previous {
       @extend %flex-wrap;
+      grid-template-columns: auto auto;
+      column-gap: 0.5em;
       .button {
         margin: 0.3em 0.2em;
         text-transform: none;
       }
+      .remove {
+        margin-left: -1.75em;
+        width: 1.75em;
+        height: 1.75em;
+        top: -1.5em;
+        right: -0.65em;
+        background-color: rgb(60, 60, 50);
```

Comment:
please use the existing `c-***` colors, and see how `mix` is used in our scss files to create new colours from them

Reply:
https://github.com/lichess-org/lila/blob/master/ui/common/css/theme/_default.scss#L27-L33